apis/v1alpha1/gatewayclass_types.go

```diff
@@ -85,10 +85,21 @@ type GatewayClassSpec struct {
 type GatewayClassConditionType string
 
 const (
-	// GatewayClassConditionStatusInvalidParameters indicates the
-	// validity of the Parameters set for a given controller. This
-	// will initially start off as "Unknown".
-	GatewayClassConditionStatusInvalidParameters GatewayClassConditionType = "InvalidParameters"
+	// GatewayClassConditionStatusAccepted indicates that the GatewayClass
```

Comment:
To follow up with the suggestion from @jpeach, it might be worth standardizing on naming between this and routes since they use `Admitted` for a similar purpose. I don't have a strong preference between the words but might slightly prefer using `Admitted` here to minimize the changes necessary.

Reply:
That makes sense. `Admitted` it is.